src/redux/actions/actionHelpers.js

```diff
@@ -0,0 +1,22 @@
+import { toast } from 'react-toastify';
+
+export const fetchOptions = (method) => ({
+  method,
+  headers: {
+    Accept: '*/*',
+    'Content-Type': 'application/json',
+    authorization: localStorage.getItem('token'),
+  },
+  mode: 'cors',
+});
+
+export const flashToast = (resp) => {
+  const res = resp.status === 200 ? 'success' : 'error';
+  return toast[res](resp.message);
+};
+
+const ref = 'https://codepirates-ah-backend.herokuapp.com/api/v1/articles';
```

Comment:
how about using `BACKEND_URL` in action type  as we are all using it

Reply:
Sure thanks, it had slipped my mind. factored in